src/Magnum/Audio/Context.h

```diff
@@ -107,6 +152,42 @@ class MAGNUM_AUDIO_EXPORT Context {
         ~Context();
 
         /**
+         * @brief Whether HRTFs (Head Related Transfer Functions) are enabled
+         *
+         * HRFTs may not be enabled/disabled in a running context. Instead
+         * create a new @ref Context with HRFTs enabled or disabled.
+         * @see @ref hrtfStatus(), @ref Audio::Context::Configuration::setHrtf(),
+         *      @fn_alc{GetIntegerv} with @def_alc{HRTF_SOFT}
+         * @requires_al_extension Extension @alc_extension{SOFTX,HRTF} or
+         *      @alc_extension{SOFT,HRTF}
+         */
+        bool isHrtfEnabled() const;
+
+        /**
+         * @brief HRTF status
+         *
+         * @see @ref isHrtfEnabled(), @fn_alc{GetIntegerv} with
+         *      @def_alc{HRTF_STATUS_SOFT}
+         * @requires_al_extension Extension @alc_extension{SOFTX,HRTF} or
+         *      @alc_extension{SOFT,HRTF}
+         */
+        HrtfStatus hrtfStatus() const;
+
+        /**
+         * @brief Hrtf specifier
+         *
+         * Name of the hrtf being used.
+         *
+         * @see @fn_al{GetString} with @def_alc{HRTF_SPECIFIER_SOFT}
+         * @requires_al_extension @alc_extension{SOFT,HRTF}
+         */
+        std::string hrtfSpecifier() const {
+            if(!Context::current()->isExtensionSupported<Extensions::ALC::SOFT::HRTF>())
```

Comment:
Personally I would omit this check. It's user responsibility to call only functions that are supported by the driver (and that fact is noted by the `@requires_al_extension` line) and OpenGL APIs behave the same (crash/whatever if you call something that is not supported by the driver).